dist/index.html

```diff
@@ -1 +1,34 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"/><meta http-equiv="X-UA-Compatible" content="IE=edge"/><meta name="viewport" content="width=device-width,initial-scale=1"/><title>mario leaderboard</title><script defer="defer" src="/webpack-demo/main.js"></script></head><body><h1>leaderboard</h1><section class="main-section"><div class="display-score"><div class="display-score-heading"><h2>Recent score</h2><button>Refresh</button></div></div><div class="input-score"><h2>Add your score</h2><form action="GET"><input placeholder="your name"/> <input placeholder="your score"/> <button>submit</button></form></div></section></body></html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>mario leaderboard</title>
+  <script defer src="/webpack-demo/main.js"></script></head>
+  <body>
+    <h1 id="h1">leaderboard</h1>
+    <section class="main-section">
+      <div class="display-score">
+        <div class="display-score-heading">
+          <h2>Recent score</h2>
+          <button id="refresh">Refresh</button>
+        </div>
+        <ul id="score-list"></ul>
+      </div>
+      <div class="input-score">
+        <h2>Add your score</h2>
+        <form id="form">
+          <input type="text" placeholder="your name" id="user-input" />
+          <input
+            type="number"
+            placeholder="your score"
+            id="score-input"
+            min="1"
+          />
+          <button>submit</button>
```

Comment:
- [ ] When I submit the new score, the score upload to the API, but this doesn't auto-refresh the score list. I need to refresh the page or something like that to see the new Score, I kindly invite you to add an auto-refresher when you submit a new score. 🐶 
- [OPTIONAL] Why not add `require` on the inputs form? 🌵